src/phoenix/db/migrate.py

```diff
@@ -1,33 +1,19 @@
 import logging
-from contextlib import contextmanager
 from pathlib import Path
 from threading import Thread
-from typing import Any, Iterator, List
 
 from alembic import command
 from alembic.config import Config
 from sqlalchemy import URL
 
+from phoenix.settings import Settings
 
-@contextmanager
-def using_log_level(new_level: int, logger_names: List[str]) -> Iterator[Any]:
-    original_levels = {}
-    try:
-        for name in logger_names:
-            logger = logging.getLogger(name)
-            # Save the original log level
-            original_levels[name] = logger.level
-            # Set the new log level
-            logger.setLevel(new_level)
-        yield
-    finally:
-        # Revert log levels to their original states
-        for name, level in original_levels.items():
-            logger = logging.getLogger(name)
-            logger.setLevel(level)
+logger = logging.getLogger(__name__)
 
 
-logger = logging.getLogger(__name__)
+def printif(condition: bool, text: str) -> None:
+    if condition:
+        print(text)
```

Comment:
This utility function exists in `src/phoenix/utilities`

Reply:
It doesn't actually, it uses tqdm which I don't want to use. I agree it should be migrated but for now I just want this.